src/cases/models/role-exclusions/role-exclusion.model.ts

```diff
@@ -0,0 +1,7 @@
+export interface RoleExclusion {
+    type: string;
+    name: string;
+    userType: string;
+    Notes: string;
```

Comment:
Should Notes be notes (i.e. lowercase n)?

Reply:
yes agree, done